src/accelerate/utils/dataclasses.py

```diff
@@ -350,6 +350,117 @@ def __post_init__(self):
                 raise ValueError(f"`optimization_level` must be one of {' or '.join(get_args(OptLevel))}")
 
 
+# Literal
+ProfilerActivity = Literal["cpu", "xpu", "mtia", "cuda"]
+
+
+@dataclass
+class ProfileKwargs(KwargsHandler):
```

Comment:
I think for this docstring to be included in the API docs, you need to add an entry to `docs/source/package_references/kwargs.md`.

Reply:
Done `06a2430`